.github/workflows/tests.yml

```diff
@@ -19,12 +17,8 @@ on:
 jobs:
   phpunit-tests:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        php: [ 7.3 ]
 
-    name: PHPUnit (PHP ${{ matrix.php }})
+    name: PHPUnit (PHP 7.3)
 
```

Comment:
FYI this could have been used to run the tests on various PHP versions (eg. 7.4 & 8.0)

Reply:
oh hmmm... yeah forward-support's a nice idea. I'll add it, see if it passes out of the box, if so I'll keep otherwise I'll leave as-is